game/src/devtools/story.rs

```diff
@@ -451,14 +451,14 @@ impl Marker {
         }
     }
 
-    fn draw_hovered(&self, g: &mut GfxCtx, app: &App) {
+    fn draw_hovered(&self, g: &mut GfxCtx) {
         let mut batch = GeomBatch::new();
         if self.pts.len() == 1 {
             batch.append(
                 GeomBatch::load_svg(g, "system/assets/timeline/goal_pos.svg")
                     .scale(2.0)
                     .centered_on(self.pts[0])
-                    .color(RewriteColor::Change(Color::hex("#5B5B5B"), app.cs.hovering)),
```

Comment:
This is an unused mode that already has a few random hardcoded colors around, so not really making it much worse.